packages/model/src/dsl.ts

```diff
@@ -244,6 +246,13 @@ export function UX<T extends Obj> (
   }
 }
 
+export function SEARCH<T extends Obj> (searchConfig: ClassSearchConfig) {
```

Comment:
Why it is Upper case?

Reply:
I looked at the @UX and it looked as a nice idea, will fix this :)